lib/ClangImporter/ImportMacro.cpp

```diff
@@ -322,15 +322,17 @@ static Optional<std::pair<llvm::APSInt, Type>>
   return None;
 }
 
-
 static ValueDecl *importMacro(ClangImporter::Implementation &impl,
-                              DeclContext *DC,
-                              Identifier name,
-                              const clang::MacroInfo *macro,
-                              ClangNode ClangN,
+                              llvm::SmallSet<StringRef, 4> &visitedMacros,
```

Comment:
I couldn't find a `SmallSetImpl` or equivalent so I think we'll just have to use the full `SmallSet<StringRef, 4>`. If you'd prefer I'm happy to add a template parameter to infer the size.

Reply:
I see other instances of such usage in llvm, I think it's fine.